src/org/labkey/test/tests/component/EditableGridTest.java

```diff
@@ -0,0 +1,99 @@
+package org.labkey.test.tests.component;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.components.glassLibrary.grids.EditableGrid;
+import org.labkey.test.pages.test.CoreComponentsTestPage;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@Category({DailyB.class})   // until there's a component suite
```

Comment:
There _is_ a component suite!
Our test runner creates one based on the package name after `tests`, e.g. `org.labkey.test.tests.`**`component`**
That being said. We want this to be in a daily suite as well.
```suggestion
@Category({DailyB.class})
```

Reply:
Nice!